### DISCUSSION TITLE
Room-Assistant Installation on Mobile Devices

### DISCUSSION THREAD

**Discussion by rickwangtw** [mKeRix/room-assistant#420] [Ideas]
Would it be possible to install room-assistant on to older phones? Older phones have access to WIFI and BT and we should be able to get access through either Android or iOS APIs.
I don't have Raspberry PIs though a couple of older Android phones, throwing this out as an idea.
Thanks!

**Comment by sj3fk3** [comment]
Could you get node.js running on the phone? or Docker?

**Reply by muniter** [reply]
Yes using termux you can get node.js in an android phone https://wiki.termux.com/index.php?title=Node.js&mobileaction=toggle_view_mobile

Just don't see any way that termux provides for querying bluetooth devices. 

EDIT: Someone wrote some api implementation https://github.com/StevenSalazarM/Termux-api-bluetooth
